test/unittest/tests/test_unit_test_template.cpp

```diff
@@ -0,0 +1,34 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#include <catch2/catch.hpp>
+#include "wt_internal.h"
+#include "wrappers/mock_session.h"
```

Comment:
There is an issue, if we specify a directory for the unit test to be created, then the link here is wrong. It needs to be "../wrappers/mock_session.h" instead. Should I cover that case in the script?

Reply:
This is a bit painful I agree, maybe we should enforce that all new tests are part of a module subdirectory and move top level tests down? At least then the tree will get neater and it won't be unclear as to when it is valid to create a top level test vs not.

Reply:
With the template being inside the bash script, I was able to configure the include dynamically according to if the -m option is set. Please take a look.

Reply:
nice!